infrastructure/zk/src/database.ts

```diff
@@ -140,25 +140,6 @@ export async function setupForDal(dalPath: DalPath, dbUrl: string) {
 }
 
 export async function setup(opts: DbOpts) {
-    if (process.env.TEMPLATE_DATABASE_URL) {
```

Comment:
Dumb question: Is the `TEMPLATE_DATABASE_URL` functionality not useful? Personally, it does look somewhat unintuitive, but it could be useful in some cases. As a possible alternative, it could be possible to remove `template_database_url` from the EN config, but still retain the functionality here. If you do remove `template_database_url` from the EN config, please remove [its mention in the snapshot recovery test](https://github.com/matter-labs/zksync-era/blob/0adab9ea84412c62d46893cc5b66368b2294d760/core/tests/snapshot-recovery-test/tests/snapshot-recovery.test.ts#L192).

Reply:
It definitely doesn't make sense to maintain it if restoring db from snapshot requires tweaking its content  - it is too fragile. We can reintroduce it if a need arises.